src/player/HTML5Player/HTML5Player.ts

```diff
@@ -30,6 +30,10 @@ export class HTML5Player extends Player {
     this.mediaElement.addEventListener('playing', () => {
       this.emit(Events.PLAYER_STATE_PLAYING);
     });
+    
+    this.mediaElement.addEventListener('pause', () => {
```

Comment:
Any particular reason what this would fix? Or was there an issue with pause to begin with?

The reason pause is bound to the instruction rather than the event is so that ad plugins can pause while ignoring setting the state, although this feature has not been implemented yet. If an ad starts, it merely means the content paused but not the ad, thus the facade should not indicate a paused state (eventhough the content is paused, ads might not be).

Pseudo code would be something like this:
```javascript
pause({ byUserGesture }) {
  mediaElement.pause();

  // If not by a user gesture, we can skip this, meaning it's a plugin that 
  // pauses the content in order to display something else (eg: ads)
  if (byUserGesture) {
    emit event -> Events.PAUSE
  }
}
```